src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+            pipeline: [{$project: {divide: {$divide: ["$int", "$int2"]}}}]
+            cursor: {}
+
+- Name: ExpressionExp
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionExp
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {exp: {$exp: "$int"}}}]
+            cursor: {}
+
+- Name: ExpressionFilter
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionFilter
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {filter: { $filter: {
+              input: "$nestedObj",
+              as: "a",
+              cond: { $gte: ["$$a.int", 0] }}}}}]
+            cursor: {}
+
+- Name: ExpressionFloor
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionFloor
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {floor: { $floor: $double}}}]
+            cursor: {}
+
+- Name: ExpressionIfNull
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionIfNull
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {ifNull: { $ifNull: [$nullVal, 0]}}}]
+            cursor: {}
+
+- Name: ExpressionIndexOfBytes
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionIndexOfBytes
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {idexOfBytes: { $indexOfBytes: [$string, "b"]}}}]
```

Comment:
s/idexOfBytes/indexOfBytes

Also, if we want to get an index deterministically (that is, return the same index everytime), then we shouldn't be using $string as the first argument.

Similar comment for indexOfCP

Reply:
Done.